File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/ClientAccountModel.ts
import { ClientModel } from "./clientModel";

export interface ClientAccountModel {
    clientAccountId: number;
    clientAccountCode: string;
    clientAccountNameAr: string;
    clientAccountNameEn: string;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    icon: string;
    color: string;
    displayOrder: number | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

}

File: SofiForce.Models/Models/EntityModels/ClientAccountModel.cs
using System;

namespace Models
{
    public class ClientAccountModel
    {
        public int ClientAccountId { get; set; }
        public string ClientAccountCode { get; set; }
        public string ClientAccountNameAr { get; set; }
        public string ClientAccountNameEn { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public int? DisplayOrder { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}
